src/servers/ZoneServer/zonepackethandlers.ts

```diff
@@ -1586,6 +1691,22 @@ const packetHandlers: any = {
       )
     ) {
       const { characterId: vehicleGuid } = vehicleToMount.npcData;
+      const { modelId: vehicleTypeId } = vehicleToMount.npcData;
+      switch (vehicleTypeId) {
+        case 1:
+          client.mountedVehicleType = "offroader";
+          break;
+        case 2:
+          client.mountedVehicleType = "pickup";
+          break;
+        case 3:
+          client.mountedVehicleType = "policecar";
+          client.isPolice = true;
```

Comment:
it's kinda a duplication ? like if you wanna check if "isPolice" you can already check like that `if client.mountedVehicleType === policecar`